src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +53,36 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        User bob = new User();
+        bob.setEmail("bob@gmail.com");
+        bob.setPassword("12345");
```

Comment:
The `User` object is being instantiated directly. If your project uses dependency injection, consider obtaining the `User` instance through the injector to maintain consistency with the rest of the code.